tests/integration/test_envelope.py

```diff
@@ -12,3 +17,91 @@ def test_envelope(mini_sentry, relay_chain):
     event = mini_sentry.captured_events.get(timeout=1).get_event()
 
     assert event["logentry"] == {"formatted": "Hello, World!"}
+
+
+def test_measure_items_envelope(mini_sentry, relay_chain):
```

Comment:
This is failing for the chain `relay->relay->sentry`.

Reply:
fixed in https://github.com/getsentry/relay/pull/724/commits/a04ca53acb2dc33465513ba69d398ffcecc2238c